README.md

```diff
@@ -62,17 +62,18 @@ In your pom.xml, add:
 
 ## Config
 
-If you're running Neo4j on localhost with the default ports, you only have to configure your password in `spark.neo4j.bolt.password=<password>`.
```

Comment:
can we highlight prominently that the config has changed?
do we still support the old config as a fallback?

Reply:
Should we support the old config?

Reply:
I added this: `**Nb.**  The old spark config prefix `spark.neo4j.bolt` is deprecated and will be removed in the next release.`